test/jasmine/tests/scatterpolargl_test.js

```diff
@@ -39,8 +39,8 @@ describe('Test scatterpolargl hover:', function() {
 
     [{
         desc: 'base',
-        nums: 'r: 2.920135\nθ: 138.2489°',
-        name: 'Trial 4'
+        nums: 'r: 3.886013\nθ: 125.2822°',
+        name: 'Trial 3'
```

Comment:
This (and below) is a weird one - as far as I can tell the new point is in fact the closest to the cursor, and the old one is a couple of pixels farther away, but I don't know what was making us pick the other one previously. I will note however that `scattergl` (which this inherits from) and `scatter` use different distance functions (mostly having to do with special handling when you're inside two markers, to prioritize the smaller marker), I didn't try to harmonize that here but we probably should at some point.